spec/javascripts/helpers/test-coverage.mjs

```diff
@@ -0,0 +1,9 @@
+window.__coverage__ ??= {};
+
+class TestCoverageReporter {
+  static suiteDone(result) {
+    jasmine.getEnv().setSuiteProperty("__coverage__", window.__coverage__);
```

Comment:
Could we add a code comment here eg. `Transfer the property from the browser to Node`

Reply:
Done.